src/main/java/net/neoforged/moddevgradle/internal/PrepareRunOrTest.java

```diff
@@ -104,11 +104,19 @@ abstract class PrepareRunOrTest extends DefaultTask {
     @Optional
     public abstract Property<VersionCapabilities> getVersionCapabilities();
 
+    /**
+     * The property that decides whether DevLogin is enabled.
+     */
+    @Input
+    @Optional
```

Comment:
Choose between optional and convention false.

Reply:
I think we already had this conversation before, the property could be forced to an empty provider. In any case, the other conventioned property (the version capability) has optional too, so for consistency I'm adding it here too.

Reply:
I think it's confusing anyway. Worst case we fix both.